packages/framework/src/Resources/translations/messages.en.po

```diff
@@ -3958,7 +3955,7 @@ msgstr ""
 msgid "Weight (g)"
 msgstr ""
 
-msgid "Weight limit"
+msgid "Weight limit (optional)"
```

Comment:
Will it be clear enough what this field means? 
It's maximum weight to apply this price, right? Will it be understandable for admin?

Reply:
what about this, better? :slightly_smiling_face: e40b15dd2a